tests/api.c

```diff
@@ -19712,6 +19712,103 @@ static int test_wc_RsaPublicKeyDecodeRaw(void)
 } /* END test_wc_RsaPublicKeyDecodeRaw */
 
 
+/*
+ * Testing wc_RsaPrivateKeyDecodeRaw()
+ */
+static int test_wc_RsaPrivateKeyDecodeRaw(void)
+{
+    EXPECT_DECLS;
+#if !defined(NO_RSA) && !defined(WOLFSSL_RSA_PUBLIC_ONLY)
```

Comment:
Looks like this also needs wrapped with !HAVE_SELFTEST. 

In file included from tests/api.c:152:
tests/api.c: In function ‘test_wc_RsaPrivateKeyDecodeRaw’:
tests/api.c:19733:17: error: implicit declaration of function ‘wc_RsaPrivateKeyDecodeRaw’; did you mean ‘wc_RsaPrivateKeyDecode’? [-Werror=implicit-function-declaration]
19733 |     ExpectIntEQ(wc_RsaPrivateKeyDecodeRaw(&n, sizeof(n),
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~
./tests/unit.h:160:24: note: in definition of macro ‘ExpectInt’
  160 |         int _x = (int)(x);                                                     \
      |                        ^
tests/api.c:19733:5: note: in expansion of macro ‘ExpectIntEQ’
19733 |     ExpectIntEQ(wc_RsaPrivateKeyDecodeRaw(&n, sizeof(n),
      |     ^~~~~~~~~~~
tests/api.c:19733:17: error: nested extern declaration of ‘wc_RsaPrivateKeyDecodeRaw’ [-Werror=nested-externs]
19733 |     ExpectIntEQ(wc_RsaPrivateKeyDecodeRaw(&n, sizeof(n),
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~
./tests/unit.h:160:24: note: in definition of macro ‘ExpectInt’
  160 |         int _x = (int)(x);                                                     \
      |                        ^
tests/api.c:19733:5: note: in expansion of macro ‘ExpectIntEQ’
19733 |     ExpectIntEQ(wc_RsaPrivateKeyDecodeRaw(&n, sizeof(n),
      |     ^~~~~~~~~~~